src/main/com/example006/radek/Operation.java

```diff
@@ -0,0 +1,8 @@
+package main.com.example006.radek;
+
+public enum Operation {
+    PLUS,
```

Comment:
v tom Enumu by to tez chtelo odsadit jeden radek za nazvem neumu, ale nechem tak uz